### PR TITLE
Html and css files changed for Desktop version

### PR DESCRIPTION
**Pull request by Alibaba2023**
# In this milestone, I present the following changes:
- Desktop version of the portfolio 
- Responsive version in min-width (786px)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -366,17 +396,21 @@ <h3 class="heading-tertiary heading-feature">
...
+                  need coded, don’t hesitate to contact me.
+                </p>
+                <a href="#" class="btn btn-inline-block">
+                  <p>Get my resume</p>
+                </a>
+              </div>
+            </div>
           </div>
         </div>
         <div class="skills">
```
Comment (NahnahAJ) [inline index.html:416]: - [ ] Please ensure that the titles of each of the skills are all in a straight line as seen in the Figma Design :+1: 
- Your Design
image

- The Figma Design
image

File: index.html [hunk 2]
```diff
@@ -438,7 +472,7 @@ <h4 class="skill-header flex">
           <form
             action="https://formspree.io/f/xrgvjkzj"
             method="post"
-            class="contact-form flex contact-container"
+            class="contact-form flex"
           >
             <input
               class="contact-input"
```
Comment (NahnahAJ) [inline index.html:478]: - [ ] In the contact section, the textarea isn't styled as seen in the Figma design so please fix this :100: 
- Your Design
image

- The Figma Design
image

- [ ] Lastly, I noticed that your buttons don't have the right button interaction styles. So kindly refer to the Figma design and ensure that the right button interaction styles are applied to all of your buttons. :100: 
- The Figma Design
image
- Your Design on Hover
image

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @Alibaba2023 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Professional README :heavy_check_mark: 
- No linter errors :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @Alibaba2023, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good job on implementing the changes ✔️ 
## Optional suggestions
- [ ] I would suggest you please use `background: #FBF8F7;` to your input field of your text area. This is the small changes so I am giving it an optional.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**